### PR TITLE
Remove validation note as already existing in ocvalidate Readme

### PR DESCRIPTION
**Pull request by perez987**
Removed ocvalidate validation note as duplicate. I am not sure if source code note is ok or not.
Thanks @PMheart for your comments.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ OpenCore Changelog
 ==================
 #### v0.8.1
 - Improved `ExtendBTFeatureFlags` quirk on newer macOS versions, thx @lvs1974
+- Source code: codestyle fixes, cleanup and conversions
```
Comment (PMheart) [inline Changelog.md:5]: Please remove this as well. Thanks.

File: Changelog.md [hunk 2]
```diff
@@ -2,6 +2,9 @@ OpenCore Changelog
 ==================
 #### v0.8.1
 - Improved `ExtendBTFeatureFlags` quirk on newer macOS versions, thx @lvs1974
+- Added notes about DMAR table and `ForceAquantiaEthernet`, thx @kokowski
+- Added System option in `LauncherOption` property, thx @stevezhengshiqi
```
Comment (PMheart) [inline Changelog.md:6]: Please drop this.

Comment (perez987) [reply Changelog.md:6]: Done.

**Comment by PMheart** [comment]
Thanks. Merged.

**Comment by perez987** [comment]
Thanks @PMheart to you for your patience. I was trying to help doing my first PR as novice.

**Pull request merged**